### PR TITLE
stats data

### PR DESCRIPTION
**Pull request by anubra266**
Closes EveripediaNetwork/issues#570
CleanShot 2022-07-26 at 01 55 11@2x

### CODE REVIEW

**Comment by kesar** [comment]
missing icons

**Comment by anubra266** [comment]
@Simplypheyie96 I need the other Icons used [here](https://everipedia.org/iq/stats) as svg. If you could put them in the figma board...that'll be great

**Comment by Simplypheyie96** [comment]
> @Simplypheyie96 I need the other Icons used [here](https://everipedia.org/iq/stats) as svg. If you could put them in the figma board...that'll be great

check in with gabriel. he designed the page

**Review by Simplypheyie96 — COMMENTED** [review]
some icons are missing here @anubra266

**Comment by anubra266** [comment]
> some icons are missing here @anubra266 

Yeah @Simplypheyie96 its work in progress, I told you I'm yet to get the svgs. I already reached out to Gabriel. He said he'll reach out to kesar and get back.

File: src/pages/dashboard/stats.tsx [hunk 1]
```diff
@@ -1,34 +1,97 @@
...
+    },
+    {
+      label: 'LP liquidity SushiSwap',
+      value: data.lp?.sushiswap,
+      icon: UniswapEllipse,
+    },
+    { label: 'LP liquidity QuickSwap', value: data.lp?.quickswap },
+  ]
+  const apps = [
+    { label: 'PredIQt markets', value: data.prediqt?.markets },
```
Comment (kesar) [inline src/pages/dashboard/stats.tsx:77]:
```suggestion
```

**Comment by kesar** [comment]
what's the status of this? PR has been paused for a week, is there anything missing / needed?

**Comment by anubra266** [comment]
> what's the status of this? PR has been paused for a week, is there anything missing / needed?

@kesar Just the icons are missing. Will try to get them myself and wrap it up.

**Comment by kesar** [comment]
> > what's the status of this? PR has been paused for a week, is there anything missing / needed?
> 
> @kesar Just the icons are missing. Will try to get them myself and wrap it up.

ok or get help from Gabriel

**Review by kesar — APPROVED** [review]

**Pull request merged**